core/sr-primitives/src/generic/digest.rs

```diff
@@ -71,14 +71,19 @@ pub enum DigestItem<Hash, AuthorityId, SealSignature> {
 	/// block. It is created for every block iff runtime supports changes
 	/// trie creation.
 	ChangesTrieRoot(Hash),
-	/// The old way to put a Seal on it.  Deprecated.
-	#[deprecated(
-		since = "1.0",
-		note = "New versions of Substrate will never generate this, and it will be rejected on new blockchains.",
-	)]
-	Seal(u64, SealSignature),
-	/// Put a Seal on it
+	/// A message from the runtime to the consensus engine. This should *never*
+	/// be generated by the native code of any consensus engine, but this is not
+	/// checked (yet).
 	Consensus(ConsensusEngineId, Vec<u8>),
+	/// Put a Seal on it.
+	Seal(ConsensusEngineId, SealSignature),
```

Comment:
In general, I'd expect that `SealSignature` is a `Vec<u8>`, right? The main reason being that we may swap out consensus engines mid-flight. 

But I'm fine leaving it as a generic param so that chains which opt-out of that feature can have a more compact representation and quicker decoding. BABE should implement `CompatibleDigestItem` for both.

Reply:
Done.